puppet/modules/web/files/rpm/robots.txt

```diff
@@ -0,0 +1,3 @@
+User-agent: *
+Disallow: /foreman/nightly/
```

Comment:
```suggestion
Disallow: /candlepin/nightly/
Disallow: /foreman/nightly/
```

Reply:
We phased out candlepin nightly for now.